prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_OML_20230831-0001.hl7

```diff
@@ -1,5 +1,5 @@
 MSH|^~\&|Epic^1.2.840.114350.1.13.145.2.7.2.695071^ISO|Centracare^centracare.com^DNS|NATUS^natus.health.state.mn.us^DNS|MN Public Health Lab^2.16.840.1.114222.4.1.10080^ISO|20230506102916+0000||OML^O21^OML_O21|31808297|P|2.5.1|||AL|AL|USA|UNICODE UTF-8|||LAB_PRU_COMPONENT^^2.16.840.1.113883.9.82^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
-PID|1||11102779^^^^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S^^^^L|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^IG^56527-9657^USA^H^^OTTER TAIL||^PRN^PH^^^^^^(763)555-5555^^^(763) 555 555|||||||||||N||||||N
+PID|1||11102779^^^^MR||SMITH^BB SARAH^^^^^L|SMITH^SADIE^S|20230504131023-0500|M|||555 STATE HIGHWAY 13^^DEER CREEK^IG^56527-9657^USA^H^^OTTER TAIL||^PRN^PH^^^^^^(763)555-5555^^^(763) 555 555|||||||||||N||||||N
```

Comment:
There were slight changes in behavior here, mostly related to removal of default values for some fields. It wasn't clear when to remove the field from the expected output or to add more detail to the expected input, so hoping somebody from the TI team can advise, thanks!

Reply:
We were also questioning this value. From our side it appeared to be hardcoded from what I remember, but I will also wait for someone else to chime in; However - from what I do know at this time - we weren't requiring PID-5.7

Reply:
As discussed in daily scrum, we're ok with the empty PID-6.7